internal/database/actions.go

```diff
@@ -254,7 +253,7 @@ func CreateMeasurement(sess db.Session, reportID sql.NullString, testName string
 func CreateResult(sess db.Session, homePath string, testGroupName string, networkID int64) (*Result, error) {
 	startTime := time.Now().UTC()
 
-	p, err := utils.MakeResultsDir(homePath, testGroupName, startTime)
+	p, err := makeResultsDir(homePath, testGroupName, startTime)
```

Comment:
If `utils.MakeResultsDir` is now unused, can you remove its implementation form the codebase?